CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+## dev
+
+* `aframe-core` merged with `aframe`. `window.AFRAME` exposes what `aframe-core` was
```

Comment:
sorry, I mean using tag commit messages to store this info. instead of a separate `CHANGELOG.md` file. but this is good too.